src/SearchResult.js

```diff
@@ -1,11 +1,12 @@
 /**
- *
  * @param {Collection} dataCollection
  * @param {int} total
  * @param {Document[]} documents
  * @param {object} [aggregations]
  * @param {object} [searchArgs]
  * @param {KuzzleSearchResult} [previous]
+ * @property {Collection} dataCollection
```

Comment:
This states the object built the constructor function has a defined property called dataCollection of type Collection